test/Testing/LeanCode.IntegrationTestHelpers.Tests/IntegrationTestsConfigurationTests.cs

```diff
@@ -20,8 +20,8 @@ public void Custom_configuration_keys_are_respected()
 
         var config = new ConfigurationBuilder().Add(configurationOverrides).Build();
 
-        Assert.NotEmpty(config.GetValue<string>(customConnectionStringKey));
-        Assert.NotEmpty(config.GetValue<string>(customInternalBaseKey));
-        Assert.NotEmpty(config.GetValue<string>(customPublicBaseKey));
+        Assert.NotEmpty(config.GetValue<string>(customConnectionStringKey) ?? string.Empty);
+        Assert.NotEmpty(config.GetValue<string>(customInternalBaseKey) ?? string.Empty);
+        Assert.NotEmpty(config.GetValue<string>(customPublicBaseKey) ?? string.Empty);
```

Comment:
```suggestion
        Assert.NotEmpty(config.GetValue<string>(customConnectionStringKey)!);
        Assert.NotEmpty(config.GetValue<string>(customInternalBaseKey)!);
        Assert.NotEmpty(config.GetValue<string>(customPublicBaseKey)!);
```

Same here - these are tests, they make _some_ assumptions.